projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -34,10 +36,26 @@ export class TabParagraphContainerComponent
 
   subscription: Subscription;
 
+  constructor(
+    componentData: CmsComponentData<CMSTabParagraphContainer>,
+    cmsService: CmsService,
+    winRef: WindowRef,
+    // tslint:disable-next-line:unified-signatures
+    breakpointService?: BreakpointService
+  );
+  /**
+   * @deprecated since 2.1
+   */
+  constructor(
+    componentData: CmsComponentData<CMSTabParagraphContainer>,
+    cmsService: CmsService,
+    winRef: WindowRef
+  );
   constructor(
     public componentData: CmsComponentData<CMSTabParagraphContainer>,
     protected cmsService: CmsService,
-    protected winRef: WindowRef
+    protected winRef: WindowRef,
```

Comment:
`winRef` is a new dependency in this constructor, so for backward compatibility (as this PR is going to land in minor version `2.1`), it should be also optional.

Reply:
I'm sorry. My bad! It was an already required parameter, so no need to treat it special.